### ISSUE TITLE
Packages not installing correctly

### ISSUE THREAD

**Issue by bippuspm** [IBIEM/community#14]
When I try to install new packages (multcomp and agricolae) either by clicking "install" in the packages pane or by the "install.packages" function, it comes up with a couple warnings and the message ends with "The downloaded source packages are in ‘/tmp/Rtmp4RIwpW/downloaded_packages’". 

Some of the error messages I'm getting are:
Error : package ‘mvtnorm’ 1.0.6 was found, but >= 1.0.10 is required by ‘multcomp’
ERROR: lazy loading failed for package ‘multcomp’
ERROR: configuration failed for package ‘units’
ERROR: dependency ‘units’ is not available for package ‘sf’
ERROR: dependency ‘sf’ is not available for package ‘spdep’

Do you know what could be happening here?

**Comment by granek** [comment]
Wanted to update you that I *am* working on this.  Sorry for not responding sooner, I have been working on it for the past two days, and  I keep thinking that I am about to get it to work when something else pops up.

I'll post an update once I get both installed

## TLDR
I generally like to install libraries in the container image instead of doing a local install (which is what happens when we run install.packages inside our IBIEM RStudio instances), because it is more reproducible (e.g. other members of your project group would *also* need to install local copies of these libraries to run notebooks that depend on them).  Also, sometimes operating system libraries need to be installed, which none of us can do within RStudio (this is the case for agricolae).  

Unfortunately rebuilding the image can take a long time - more than one hour with the last change I made, so it takes a while to find out whether the change I made works.

I managed to get multcomp installed on the *test* image, but I am waiting to update the production image (what you actually work in) until I get agricolae working. Updating the the production image requires restarting everyone's containers so I try to minimize the disruption.

multcomp depends on a newer version of mvtnorm than is currently installed, but the newest version of mvtnorm depends on a newer version of R than we are using, so I had to manually find a newer version of mvtnorm that made multcomp happy but would work on our version of R.

agricolae depends on an operating system library and several other R libraries, at least one of which depends on another R library, which itself depends on an operating system library.

If you care, you can see [here](https://gitlab.oit.duke.edu/mccahill/IBIEM2018/activity) what I have been doing to get these things installed.

**Comment by granek** [comment]
OK, these are installed now!
